Dockerfile

```diff
@@ -1,9 +1,25 @@
-FROM alpine
+# Use standard golang build
+FROM golang:1.8.3-alpine
 
-RUN apk -U add ca-certificates
+# Copy our app files
+ADD . "$GOPATH/src/github.com/OrfeasZ/ecr-token-refresh"
```

Comment:
Actually, now that I'm looking at it, it doesn't matter. This only defines the build directory, it doesn't actually perform any git-bound operations. I'll change it into something more generic regardless.